rails-snow-queen/app/views/quote_mailer/new_quote.html.erb

```diff
@@ -10,11 +10,11 @@
   <dt>Address:</dt>
   <dd><%= @quote.address %></dd>
   <dt>Area to be serviced:</dt>
-  <dd><%= @quote.area.toFixed(0) %>sq ft</dd>
+  <dd><%= number_with_precision(@quote.area, precision: 0) %> sq ft</dd>
```

Comment:
You probably want to add `delimiter: ','` or `locale: 'en-ca'`

Reply:
done